app/client/src/components/pages/StateTribal.js

```diff
@@ -165,30 +165,37 @@ const searchContainerStyles = css`
   }
 `;
 
-const searchSourceButtonStyles = css`
-  height: 38px;
-  border-top-left-radius: 4px;
-  border-bottom-left-radius: 4px;
-`;
+// TODO-Tribal - Uncomment the block below
+// const searchSourceButtonStyles = css`
+//   height: 38px;
+//   border-top-left-radius: 4px;
+//   border-bottom-left-radius: 4px;
+// `;
 
 function StateTribal() {
   const location = useLocation();
   const navigate = useNavigate();
-  const organizations = useOrganizationsContext();
+  // const organizations = useOrganizationsContext(); // TODO-Tribal - uncomment this
   const services = useServicesContext();
-  const tribeMapping = useTribeMappingContext();
+  // const tribeMapping = useTribeMappingContext(); // TODO-Tribal - uncomment this
 
   // redirect to '/stateandtribe' if the url is /state or /tribe
   useEffect(() => {
     const pathname = window.location.pathname.toLowerCase();
-    if (['/state', '/state/', '/tribe', '/tribe/'].includes(pathname)) {
-      navigate('/state-and-tribal', { replace: true });
+    // TODO-Tribal - Uncomment lines below and remove the 3 state lines
+    // if (['/state', '/state/', '/tribe', '/tribe/'].includes(pathname)) {
+    //   navigate('/state-and-tribal', { replace: true });
+    // }
+    if (['/state', '/state/'].includes(pathname)) {
+      navigate('/state', { replace: true });
     }
   }, [navigate]);
```

Comment:
I don't think we need this block at all right now, but it doesn't hurt and it will be back soon so it's fine as is.